python/tools/encfs_aes_getpass.py

```diff
@@ -7,35 +7,39 @@
 encfs --standard --extpass=./encfs_aes_getpass.py ~/.crypt ~/crypt
 """
 
+import hashlib
+import json
 import os
 import sys
-import json
-import hashlib
+from typing import TYPE_CHECKING, Sequence
 
 import trezorlib
+import trezorlib.misc
+from trezorlib.client import TrezorClient
+from trezorlib.tools import Address
+from trezorlib.transport import enumerate_devices
+from trezorlib.ui import ClickUI
 
 version_tuple = tuple(map(int, trezorlib.__version__.split(".")))
 if not (0, 11) <= version_tuple < (0, 12):
     raise RuntimeError("trezorlib version mismatch (0.11.x is required)")
 
-from trezorlib.client import TrezorClient
-from trezorlib.transport import enumerate_devices
-from trezorlib.ui import ClickUI
 
-import trezorlib.misc
+if TYPE_CHECKING:
+    from trezorlib.transport import Transport
 
 
-def wait_for_devices():
-    devices = enumerate_devices()
+def wait_for_devices() -> Sequence["Transport"]:
+    devices = tuple(enumerate_devices())
```

Comment:
`enumerate_devices()` is returning an `Iterable` (inside it is a `List` however), but we need to return `Sequence` here, as the `devices` are used in `choose_device()` and it is checking its `len()`, so an `Iterable` cannot be used here.

We could change `enumerate_devices()` to return `Sequence` (or even a `List`, because it is really a list internally), but it is used on many places and it really expects `Iterable`

Reply:
I tried to change to `Sequence` and it doesn't fail for me anywhere else...?

Reply:
Nice, I did not try that, I supposed it would fail. But it is right that where `Iterable` can be used, also `Sequence` can. Changed in 2b17ea6b4